cli/tauri.js/test/jest/fixtures/app/src-tauri/src/main.rs

```diff
@@ -16,11 +16,13 @@ fn main() {
         )
         .unwrap();
       });
-      webview.eval("window.onTauriInit && window.onTauriInit()");
+      let _ = webview.dispatch(|w| {
+        w.eval("window.onTauriInit && window.onTauriInit()");
+      });
```

Comment:
I look around all `dispatch` calls and they all simpy calling `eval`.
Is it possible we just call `eval` instead?

Reply:
there's no "eval" on WebviewMut, so the dispatch is needed.